x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/defense_evasion_suspicious_certutil_commands.json

```diff
@@ -52,5 +52,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 11
+  "version": 12
```

Comment:
As above -- is this an unnecessary version bump? Or was it missed in the previous batch of 8.2 updates?